### PR TITLE
Terminal view added to /docs

### PR DESCRIPTION
**Pull request by fulyauluturk**

### CODE REVIEW

**Review by otisg — CHANGES_REQUESTED** [review]

File: docs/logs/reports-and-components.md [hunk 1]
```diff
@@ -54,12 +54,28 @@ Use them together with out-of-the-box metrics, from a Monitoring App, and you wi
...
+
+Additionally, you can expand each log to view its details in either Table or JSON format, providing you with easy access to the full log event payload, ability to add additional field value-based filters, copy field values, etc.
+
+![Sematext Cloud Logs Guide - Logs Table Details Format](../images/guide/logs/logs-table-details-format.gif) 
+
+Furthermore, you have the option to switch to **Terminal View**, which allows you to view your logs in a more condensed and compact format.
+
+![Sematext Cloud Logs Guide - Logs Table Terminal View](../images/guide/logs/logs-table-terminal-view.png) 
+
+See [Logs Table Terminal View](https://sematext.com/docs/logs/logs-table-terminal-view/) for more details.
```
[the remark this answers is not in the document]
Comment (fulyauluturk) [reply docs/logs/reports-and-components.md:71]: Changed the page name to Logs Terminal View.

File: mkdocs.yml [hunk 2]
```diff
@@ -20,6 +20,7 @@ pages:
       - Quick Start: logs/quick-start.md
       - Settings: logs/settings.md
       - Reports & Components: logs/reports-and-components.md
+      - Logs Table Terminal View: logs/logs-table-terminal-view.md
       - Logs Table Quick Actions: logs/logs-table-quick-actions.md
```
[the remark this answers is not in the document]
Comment (fulyauluturk) [reply mkdocs.yml:24]: Quick actions are only available in LT for now.

[the remark this answers is not in the document]
Comment (otisg) [reply mkdocs.yml:24]: Does logs-table-termina-view.md make sense?  Is this page about both LT and TV?

**Comment by fulyauluturk** [comment]
@otisg Updated the page name.

**Pull request merged**